package/MDAnalysis/analysis/hbonds/hbond_analysis.py

```diff
@@ -308,6 +321,12 @@ class HydrogenBondAnalysis_OtherFF(HydrogenBondAnalysis):
 
    .. automethod:: _get_bonded_hydrogens_list
 
+   .. versionchanged:: 0.15.0
```

Comment:
Or perhaps rather use internally 0-based from now on and just state this here using `.. versionchanged`.